src/numericAlphabeticTranslator/RunManager.java

```diff
@@ -0,0 +1,15 @@
+package numericAlphabeticTranslator;
+
+public class RunManager {
+   // private String numName;
+    private int num;
+    TranslatorIOManager userInputs = new TranslatorIOManager();
```

Comment:
This is always a point of interest..... it the variablename doesn't reflect the name of the class definition in any way, is the nameing correct for the class? does it have different responsibilities?

Reply:
will do better with naming next time :)